core/src/commonTest/kotlin/com/sourcepoint/mobile_core/network/SourcepointClientTest.kt

```diff
@@ -231,4 +235,35 @@ class SourcepointClientTest {
         assertNotNull(mockEngine.requestHistory.firstOrNull { it.url.pathSegments.contains("meta-data") })
         assertNotNull(mockEngine.requestHistory.firstOrNull { it.url.pathSegments.contains("custom-metrics") })
     }
+
+    @Test
+    fun checkUrlBuilderCustomConsent() = runTest {
+        val mockEngine = mock()
+        SourcepointClient(accountId, propertyId, propertyName, httpEngine = mockEngine)
+            .customConsentGDPR(CustomConsentRequest("uuid",propertyId, emptyList(), emptyList(), emptyList()))
+        val defaultRequest = DefaultRequest()
+        assertEquals(mockEngine.requestHistory.last().url, Url("https://cdn.privacy-mgmt.com/wrapper/tcfv2/v1/gdpr/custom-consent?env="+defaultRequest.env+"&scriptType="+defaultRequest.scriptType+"&scriptVersion="+defaultRequest.scriptVersion))
+
+        SourcepointClient(accountId, propertyId, propertyName, httpEngine = mockEngine)
+            .deleteCustomConsentGDPR(CustomConsentRequest("uuid",propertyId, emptyList(), emptyList(), emptyList()))
+        assertEquals(mockEngine.requestHistory.last().url, Url("https://cdn.privacy-mgmt.com/consent/tcfv2/consent/v3/custom/16893?consentUUID=uuid"))
+    }
+
+    @Test
+    fun checkCustomConsent() = runTest {
+        val vendors = listOf("5ff4d000a228633ac048be41")
+        val categories = listOf("608bad95d08d3112188e0e36", "608bad95d08d3112188e0e2f")
+        val responseCustom = SourcepointClient(accountId, propertyId, propertyName)
+            .customConsentGDPR(CustomConsentRequest("uuid_36",propertyId, vendors, categories, emptyList()))
+        val responseDeleteCustom = SourcepointClient(accountId, propertyId, propertyName)
+            .deleteCustomConsentGDPR(CustomConsentRequest("uuid_36",propertyId, vendors, categories, emptyList()))
+
```

Comment:
@Nevazhnovu you don't need to instantiate `SourcepointClient` again, because we already have it instantiated in the top of the file (the `api` variable).

Also, when passing parameters to functions, if the function has more than 1 or 2 params, I find it easier to name it:
```diff
- api.doSomething(a, b, c)
+ api.doSomething(
+  firstParam = a,
+  second = b,
+  third = c
+ )
```
This way it's a bit more readable.